### ISSUE TITLE
Proxy support for phantomjs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#95]
```
Attached is modification for phantomjs proxy support.
```

Original issue reported on code.google.com by `floatclo...@gmail.com` on 17 Sep 2014 at 8:39

**Comment by GoogleCodeExporter** [comment]
Original comment by `floatclo...@gmail.com` on 18 Sep 2014 at 6:19

Attachments:
- [phantomjs_proxy.patch](https://storage.googleapis.com/google-code-attachments/selenium-vba/issue-95/comment-1/phantomjs_proxy.patch)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.0.20.0
```

Original comment by `florentbr` on 18 Sep 2014 at 4:26
- Changed state: **Fixed**
